src/vivarium_gates_iv_iron/constants/data_keys.py

```diff
@@ -30,47 +30,97 @@ def log_name(self):
 POPULATION = __Population()
 
 
-# TODO - sample key group used to identify keys in model
-# For more information see the tutorial:
-# https://vivarium-inputs.readthedocs.io/en/latest/tutorials/pulling_data.html#entity-measure-data
-# class __SomeDisease(NamedTuple):
-#
-#     # Keys that will be loaded into the artifact. must have a colon type declaration
-#     SOME_DISEASE_PREVALENCE: TargetString = TargetString(
-#         "cause.some_disease.prevalence"
-#     )
-#     SOME_DISEASE_INCIDENCE_RATE: TargetString = TargetString(
-#         "cause.some_disease.incidence_rate"
-#     )
-#     SOME_DISEASE_REMISSION_RATE: TargetString = TargetString(
-#         "cause.some_disease.remission_rate"
-#     )
-#     DISABILITY_WEIGHT: TargetString = TargetString(
-#         "cause.some_disease.disability_weight"
-#     )
-#     EMR: TargetString = TargetString("cause.some_disease.excess_mortality_rate")
-#     CSMR: TargetString = TargetString(
-#         "cause.some_disease.cause_specific_mortality_rate"
-#     )
-#     RESTRICTIONS: TargetString = TargetString("cause.some_disease.restrictions")
-#
-#     # Useful keys not for the artifact - distinguished by not using the colon type declaration
-#     RAW_DISEASE_PREVALENCE = TargetString("sequela.raw_disease.prevalence")
-#     RAW_DISEASE_INCIDENCE_RATE = TargetString("sequela.raw_disease.incidence_rate")
-#
-#     @property
-#     def name(self):
-#         return "some_disease"
-#
-#     @property
-#     def log_name(self):
-#         return "some disease"
-#
-#
-# SOME_DISEASE = __SomeDisease()
+class __PregnancyOutcomes(NamedTuple):
+    # Live birth, still birth, other
+    STILLBIRTH: str = "pregnancy_outcomes.stillbirth"
+    LIVE_BIRTH: str = "pregnancy_outcomes.live_birth"
+    OTHER: str = "pregnancy_outcomes.other"  # (abortion, miscarriage, ectopic pregnancy)
+
+    @property
+    def name(self):
+        return "pregnancy_outcomes"
+
+    @property
+    def log_name(self):
+        return "pregnancy_outcomes"
+
+
+PREGNANCY_OUTCOMES = __PregnancyOutcomes()
+
+
+class __Pregnancy(NamedTuple):
+
+    # Keys that will be loaded into the artifact. must have a colon type declaration
+    PREGNANCY_PREVALENCE: TargetString = TargetString(
```

Comment:
I'd recommend removing "PREGNANCY" from these keynames, so you don't end up with calls like `PREGNANCY.PREGNANCY_PREVALENCE`  and have `PREGNANCY.PREVALENCE` instead.